changelog/unreleased/issue-6527.toml

```
@@ -0,0 +1,5 @@
+type = "a" # One of: a(dded), c(hanged), d(eprecated), r(emoved), f(ixed), s(ecurity)
+message = "New pipeline rule function for normalizing message field names."
```

Can you add the function name to the message? That helps users to find the new function. :slightly_smiling_face: 

```suggestion
message = "Add `normalize_fields` pipeline rule function for normalizing message field names."
```